en/middleware/modules_communication.md

```diff
@@ -35,8 +35,8 @@ The implementation uses 2 threads, a sending and a receiving thread. The sender
 reduces the rates of the streams if the combined bandwidth is higher than the configured rate (`-r`) or the
 physical link becomes saturated. This can be checked with `mavlink status`, see if `rate mult` is less than 1.
 
-**Careful**: some of the data is accessed and modified from both threads, so when changing code or extend the
-functionality, this needs to be take into account, in order to avoid race conditions and corrupt data.
+**Careful**: some of the data is accessed and modified from both threads, so when changing code or extending the
```

Comment:
Should be a note like this 

> **Caution** Some of the data is accessed and modified from both threads. When changing code/extending functionality care must be taken in order to avoid race conditions and corrupt data.